theme_crowdfunding/static/src/scss/bootstrap.scss

```diff
@@ -0,0 +1,44 @@
+/*!
```

Comment:
Do we really need bootstrap and all these scss ?? I think it should be included in Odoo...